Bottom: acd30037d2d689c2175d48e9dd4a3eb472908139
Top:    34d000e7caaeca566ed31c37dd9eed734b55c0d3
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 20:31:12 +0000

Disable polygon debug gui


---

diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index f6bf546..e8ef1d8 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -1480,8 +1480,10 @@ ghid_build_pcb_top_window (void)
 
   gtk_tree_view_append_column (GTK_TREE_VIEW (ghidgui->polygon_debug_tv), column);
 
+#if 0
   gtk_box_pack_end (GTK_BOX (hbox_middle),
                     ghidgui->polygon_debug_tv, TRUE, TRUE, 0);
+#endif
 
   g_signal_connect (G_OBJECT (gtk_tree_view_get_selection (GTK_TREE_VIEW (ghidgui->polygon_debug_tv))),
                     "changed",
